src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -4,693 +4,689 @@
+  - name: --agent-count
+    short-summary: Set the default number of agents for the agent pools.
+    long-summary: Note that DC/OS clusters will have 1 or 2 additional public agents.
+examples:
+  - name: Create a DCOS cluster with an existing SSH key.
+    text: |-
+        az acs create --orchestrator-type DCOS -g MyResourceGroup -n MyContainerService \\
+          --ssh-key-value /path/to/publickey
+  - name: Create a DCOS cluster with two agent pools.
+    text: |-
+        az acs create -g MyResourceGroup -n MyContainerService --agent-profiles '[ \\
+          { \\
+            "name": "agentpool1" \\
+          }, \\
+          { \\
+            "name": "agentpool2" \\
+          }]'
+  - name: Create a DCOS cluster where the second agent pool has a vmSize specified.
+    text: |-
+        az acs create -g MyResourceGroup -n MyContainerService --agent-profiles '[ \\
+          { \\
+            "name": "agentpool1" \\
+          }, \\
+          { \\
+            "name": "agentpool2", \\
+            "vmSize": "Standard_D2" \\
+          }]'
+  - name: Create a DCOS cluster with agent-profiles specified from a file.
+    text: az acs create -g MyResourceGroup -n MyContainerService --agent-profiles MyAgentProfiles.json
 """
 
 helps['acs dcos'] = """
-    type: group
-    short-summary: Commands to manage a DC/OS-orchestrated Azure Container Service.
+type: group
+short-summary: Commands to manage a DC/OS-orchestrated Azure Container Service.
 """
 
 helps['acs dcos install-cli'] = """
-    type: command
-    short-summary: Download and install the DC/OS command-line tool for a cluster.
+type: command
+short-summary: Download and install the DC/OS command-line tool for a cluster.
 """
 
-helps['acs kubernetes install-cli'] = """
-    type: command
-    short-summary: Download and install the Kubernetes command-line tool for a cluster.
+helps['acs delete'] = """
+type: command
+short-summary: Delete a container service.
 """
 
 helps['acs kubernetes'] = """
-    type: group
-    short-summary: Commands to manage a Kubernetes-orchestrated Azure Container Service.
+type: group
+short-summary: Commands to manage a Kubernetes-orchestrated Azure Container Service.
 """
 
 helps['acs kubernetes get-credentials'] = """
-    type: command
-    short-summary: Download and install credentials to access a cluster.  This command requires
-                   the same private-key used to create the cluster.
-    parameters:
-        - name: --output -o
-          type: string
-          long-summary: Credentials are always in YAML format, so this argument is effectively ignored.
+type: command
+short-summary: Download and install credentials to access a cluster.  This command requires the same private-key used to create the cluster.
+parameters:
+  - name: --output -o
+    type: string
+    long-summary: Credentials are always in YAML format, so this argument is effectively ignored.
+examples:
```

Comment:
Since the ACS service is deprecated, it may not be worth adding an example to this command.